### PR TITLE
[doc] Updated Quantized data types

### PR DESCRIPTION
**Pull request by Vissidarte-Herman**
Related issue = #

### CODE REVIEW

File: docs/lang/articles/advanced/quant.md [hunk 1]
```diff
@@ -2,81 +2,96 @@
...
-format. *Quantized integers* in Taichi adopt the same format, and can contain
-non-standard number of bits:
+Quantized integers in Taichi are represented in the [two's complement](https://en.wikipedia.org/wiki/Two's_complement) format but can contain arbitrary number of bits.
+
+- To define a 10-bit signed integer type:
 
 ```python
-i10 = ti.types.quant.int(bits=10)  # 10-bit signed (default) integer type
-u5 = ti.types.quant.int(bits=5, signed=False)  # 5-bit unsigned integer type
+i10 = ti.types.quant.int(bits=10) # `signed` is set to `True` by default
```
Comment (strongoier) [inline docs/lang/articles/advanced/quant.md:30]:
```suggestion
i10 = ti.types.quant.int(bits=10)  # `signed` is set to `True` by default
```

File: docs/lang/articles/advanced/quant.md [hunk 2]
```diff
@@ -2,81 +2,96 @@
...
 ### Quantized fixed-point numbers
 
-[Fixed-point numbers](https://en.wikipedia.org/wiki/Fixed-point_arithmetic) are
-an old way to represent real numbers. The internal representation of a fixed-point number is simply an integer, and
-its actual value equals to the integer multiplied by a predefined scaling
-factor. Based on the support for quantized integers, Taichi provides *quantized
-fixed-point numbers* as follows:
+The core idea of [fixed-point numbers](https://en.wikipedia.org/wiki/Fixed-point_arithmetic) is that, if a specific range is evenly divided into multiple scale units, then a real number within that range can be approximated and represented by multiplying the value of each scale unit by an integer number. Here's an example explaining what the *scale unit* here is: If you wish to represent a real number within [0, 100] in 10 binary bits, then each *scale unit* equals 100/2<sup>10</sup> &asymp; 0.098.
+
+Taichi allows you to define quantized fixed-point types of less than 64 bits and with an arbitrary scale unit:
```
Comment (strongoier) [inline docs/lang/articles/advanced/quant.md:43]:
```suggestion
Taichi allows you to define quantized fixed-point types of less than 64 bits and with an arbitrary scale unit.
```

File: docs/lang/articles/advanced/quant.md [hunk 3]
```diff
@@ -2,81 +2,96 @@
...
-floating-point number is composed of exponent bits, fraction bits, and a sign
-bit. There are various floating-point formats:
+A [floating-point number](https://en.wikipedia.org/wiki/Floating-point_arithmetic) comprises exponent bits, fraction bits, and a sign bit. There are various floating-point formats:
 
 ![image](../static/assets/floating-point_formats.png)
 
-In Taichi, you can define a *quantized floating-point number* with arbitrary
-combination of exponent bits and fraction bits (the sign bit is made part of
-fraction bits):
+Taichi allows you to define a *quantized floating-point number* with an arbitrary combination of exponent bits and fraction bits (the sign bit is made part of the fraction bits):
```
Comment (strongoier) [inline docs/lang/articles/advanced/quant.md:72]:
```suggestion
Taichi allows you to define a *quantized floating-point number* with an arbitrary combination of exponent bits and fraction bits (the sign bit is made part of the fraction bits).
```

File: docs/lang/articles/advanced/quant.md [hunk 4]
```diff
@@ -2,81 +2,96 @@
...
 
-All the parameters you've seen above are specifying the *storage type* of a
-quantized data type. However, most quantized data types have no native support
-on hardware, so an actual value of that quantized data type needs to convert to
-a primitive type ("*compute type*") when it is involved in computation.
+All the above-mentioned parameters specify how a quantized data type is stored in your computer. However, most quantized data types have no native support on hardware, so an actual value of that quantized data type needs to be converted to a primitive type ("*compute type*") during computation.
+
+- The default compute type for quantized integers is `ti.i32`,
+- The default compute type for quantized fixed-point numbers is `ti.f32`,
+- The default compute type for floating-point numbers is `ti.f32`.
```
Comment (strongoier) [inline docs/lang/articles/advanced/quant.md:92]:
```suggestion
- The default compute type for quantized floating-point numbers is `ti.f32`.
```

**Review by strongoier — APPROVED** [review]
Thanks!

**Comment by Vissidarte-Herman** [comment]
All requested changes have been implemented. Many thanks.

**Pull request merged**